test/erc4626schema.test.js

```diff
@@ -18,15 +18,22 @@ const schema = {
             summary: { type: "string", enum: ["safe", "unsafe"] },
             review: { type: "string", minLength: 1 },
             warnings: { type: "array", items: { type: "string" } },
-            isCompatible: { type: "string", enum: ["true", "false"] },
+            useBufferForSwaps: { type: "string", enum: ["true", "false"] },
```

Comment:
The previous entry `isCompatible` was based on passing fork-tests. This change to me reads as if it is now transitioning towards an intention. Is this accurate? If so, who will be signaling the "true" or "false" entry here? 

I initially thought that it woud be included in the changes in the review request. But If I understand the changes correctly, this is not the case. Can you elaborate here, please?

Reply:
I think it's worth sharing that I had the same feeling as MkFlow that this change seems to indicate intention. Maybe it's the variable names. If we expect people out of our team to answer these, they might become a friction. Something like: `canUseBufferForSwaps` better express possibility/compatibility instead of intention.

Reply:
> I think it's worth sharing that I had the same feeling as MkFlow that this change seems to indicate intention. Maybe it's the variable names. If we expect people out of our team to answer these, they might become a friction. Something like: `canUseBufferForSwaps` better express possibility/compatibility instead of intention.

I see. We can rename it of course.

Just saying that this flag is set by the reviewer of the ERC4626 and not by the requestor.